WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/StatsViewModel.kt

```diff
@@ -189,15 +191,11 @@ class StatsViewModel
             }
         }
 
-        if (launchedFrom == StatsLaunchedFrom.FEATURE_ANNOUNCEMENT) {
-            updateRevampedInsights()
+        if (statsRevampV2FeatureConfig.isEnabled()) {
+            appPrefsWrapper.markStatsRevampFeatureAnnouncementAsDisplayed()
```

Comment:
@ravishanker, I guess you added `updateRevampedInsights()` because "Try it" button would enable the flag.  But we decided to open the flag for all users, and "Try it" button will just navigate to stats. So, I removed it.

I markStatsRevampFeatureAnnouncementAsDisplayed once stats opened. Because the user may tap "Remind me" button, then go to stats via home or menu. We don't need to show the feature intro screen in this case.

Reply:
I'm not sure about this change

Reply:
If we'll do James' suggestion, adding all cards before tapping "Try it" button, users won't need to see the announcement again if they've already seen the new stats.
I'm leaving it as it is.